include/fast_float/float_common.h

```diff
@@ -249,19 +249,21 @@ struct adjusted_mantissa {
   }
 };
 
-// Bias so we can get the real exponent with an invalid adjusted_mantissa.
-constexpr static int32_t invalid_am_bias = -0x8000;
+template <typename = void>
+struct power_tables {
+  // Bias so we can get the real exponent with an invalid adjusted_mantissa.
+  constexpr static int32_t invalid_am_bias = -0x8000;
 
-constexpr static double powers_of_ten_double[] = {
+  constexpr static double powers_of_ten_double[] = {
```

Comment:
The powers_of_ten_* tables could be moved to the binary_format template. It would probably be better encapsulation.

Reply:
Yes, `max_mantissa_*` too. Then the two constants (`invalid_am_bias` and `constant_55555`) that don't get odr-used anyway can remain static.